apps/azure-devops/azure-devops-agent-keda/azure-devops-agent.yaml

```
@@ -7,7 +7,7 @@ spec:
   releaseName: azure-devops-agent
   values:
     scaleType: Job
-    image: hmctspublic.azurecr.io/azure-devops-agent:prod-97ab16c3-1693560380 # {"$imagepolicy": "flux-system:azure-devops-agent"}
+    image: hmctspublic.azurecr.io/azure-devops-agent:prod-97ab16c3-1693560380 # {"$imagepolicy": "flux-system:azure-devops-agent-keda"}
```

that's the name of the policy which is not changed in the file
```suggestion
    image: hmctspublic.azurecr.io/azure-devops-agent:prod-97ab16c3-1693560380 # {"$imagepolicy": "flux-system:azure-devops-agent"}
```